packages/base/src/commands/commands.ts

```diff
@@ -20,21 +20,18 @@ import { ReadonlyPartialJSONObject } from '@lumino/coreutils';
 import { Coordinate } from 'ol/coordinate';
 import { fromLonLat } from 'ol/proj';
 
-import { CommandIDs, icons } from './constants';
-import { ProcessingFormDialog } from './dialogs/ProcessingFormDialog';
-import { LayerBrowserWidget } from './dialogs/layerBrowserDialog';
-import { LayerCreationFormDialog } from './dialogs/layerCreationFormDialog';
-import { SymbologyWidget } from './dialogs/symbology/symbologyDialog';
-import { targetWithCenterIcon } from './icons';
-import keybindings from './keybindings.json';
-import {
-  getSingleSelectedLayer,
-  selectedLayerIsOfType,
-  processSelectedLayer,
-} from './processing';
-import { getGeoJSONDataFromLayerSource, downloadFile } from './tools';
-import { JupyterGISTracker } from './types';
-import { JupyterGISDocumentWidget } from './widget';
+import { CommandIDs, icons } from '../constants';
```

Comment:
Great call on moving commands stuff into a directory! Maybe we could call this file `commands/index.ts`?

If you'd like, you can use `@/src/constants` instead of a relative path, then the file can be moved more easily without changing the imports ;)